api/descriptions/tags/tag_cve_post.md

```diff
@@ -69,3 +72,29 @@ Consider the following scenarios for source and binary packages:
   Refer to the following image that displays the tagged vulnerability:
   
   ![Tagged vulnerability in CentOS](https://cdn.twistlock.com/docs/api/CentOS-Vuln-Bin-Package-CVE-tagged-Ignored-Vuln.png)
+
```

Comment:
@MorShabiPAN : Can you please review the example?

Reply:
looks good